README.md

```diff
@@ -2,32 +2,78 @@
 
 
 ## Sourcery
-Refactor Python instantly and keep your codebase maintainable.
+Instantly review and improve your Python code to keep your codebase maintainable. 
 
-## GitHub Code Reviews
+Get started with `pip install sourcery-cli` to have Sourcery review all your code locally or star this repo to have our GitHub bot review your most popular Python repo
```

Comment:
do we want this duplication of the pip install with the section just below?

Reply:
Good call - can cut